src/Microsoft.Management.Configuration.Projection/Microsoft.Management.Configuration.Projection.csproj

```diff
@@ -12,6 +12,8 @@
     !!! Remove or update this on the next Microsoft.Windows.CsWinRT package version update. !!!
     -->
     <WindowsSdkPackageVersion>10.0.26100.53</WindowsSdkPackageVersion>
+    <!-- Disables the AssemblyInfo generation override in the internal build. -->

```

Comment:
I would like the comment to say why we need this. Especially after the whole ordeal to find the magic line.